isso/js/app/templates/comment-loader.pug

```diff
@@ -1,2 +1,2 @@
-div(class='isso-comment-loader' id='isso-loader-#{comment.name}')
+div(class='isso-comment-loader' id='isso-loader-' + comment.name)
```

Comment:
Shouldn' `${}` be replaced on next line too?

Reply:
That's actually intended. Doing this is no allowed any more:
```
div(id='isso-loader-#{comment.name}')
```
See [pugjs: Attribute interpolation](https://pugjs.org/language/attributes.html#attribute-interpolation) and [pugjs: v2 migration](https://pugjs.org/api/migration-v2.html#attribute-interpolation)

However, anything outside parentheses is rendered as plain text if not using the `#{}` syntax.

This is `pug` being extremely stupid, imo, but I can't change that.

Reply:
Turns out I was wrong here, using ["buffered code"](https://pugjs.org/language/code.html#buffered-code) is appropriate.

This:
```
a(class='reply' href='#') #{i18n('comment-reply')}
```
becomes this:
```
a(class='isso-reply' href='#')
    = i18n('comment-reply')
```

Note that the additional newline and indentation is needed.
This does not work:
```
a(class='isso-reply' href='#') = i18n('comment-reply')
```

Reply:
Done